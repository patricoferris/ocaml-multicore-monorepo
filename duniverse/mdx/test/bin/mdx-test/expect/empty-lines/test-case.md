This shell block contains an empty line within a padded block:

```sh
 $ echo ''

```

```sh
  $ echo "Hello..." && echo "" && echo "world!"
```

This toplevel block contains an empty line within a padded block:

```ocaml
 # print_newline;;

```
