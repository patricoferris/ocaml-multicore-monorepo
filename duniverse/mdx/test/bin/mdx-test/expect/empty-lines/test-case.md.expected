This shell block contains an empty line within a padded block:

```sh
 $ echo ''

```

```sh
  $ echo "Hello..." && echo "" && echo "world!"
  Hello...

  world!
```

This toplevel block contains an empty line within a padded block:

```ocaml
 # print_newline;;
 - : unit -> unit = <fun>
```
